ansible/roles/wsconfig/templates/dotfiles/awsconfig.j2

```diff
@@ -3,5 +3,5 @@
 [profile tcf]
 role_arn=arn:aws:iam::213722482034:role/org-admin
```

Comment:
Because we have these AWS account numbers in the repo, we should consider taking the repo private

Reply:
okay!